### PR TITLE
fabtests: add option for additional client/server arguments

### PR DESCRIPTION
**Pull request by rwespetal**
Add flags to runfabtests.sh that set additional flags to the
client/server fabtests.

Signed-off-by: Robert Wespetal <wesper@amazon.com>

### CODE REVIEW

**Comment by rajachan** [comment]
add to whitelist

**Comment by rwespetal** [comment]
bot:aws:retest

File: fabtests/scripts/runfabtests.sh [hunk 1]
```diff
@@ -619,10 +622,12 @@ function usage {
 	errcho -e " -u\tconfigure option for complex tests"
 	errcho -e " -T\ttimeout value in seconds"
 	errcho -e " -S\tStrict mode: -FI_ENODATA, -FI_ENOSYS errors would be treated as failures instead of skipped/notrun"
+	errcho -e " -C\Additional client test arguments: Parameters to pass to client fabtests"
+	errcho -e " -A\Additional server test arguments: Parameters to pass to server fabtests"
```
Comment (shefty) [inline fabtests/scripts/runfabtests.sh]: nit: Can we use -L (for listener) or P (for passive) in place of A?

Comment (rwespetal) [reply fabtests/scripts/runfabtests.sh]: Sure, I've changed it to -L.

**Comment by rwespetal** [comment]
bot:aws:retest

**Comment by rwespetal** [comment]
@shefty - do you know how to re-trigger the LANL-CI? I think that failure may have just been a transient one.

**Comment by shefty** [comment]
@hppritcha would know how to re-trigger it.  It's similar to AWS I think

**Comment by shefty** [comment]
bot:lanl:retest

**Comment by rwespetal** [comment]
Thank you, looks like that worked.

**Pull request merged**